pkg/process/config/yaml_config.go

```diff
@@ -176,24 +178,24 @@ func (a *AgentConfig) loadSysProbeYamlConfig(path string) error {
 		a.Windows.DriverBufferSize = driverBufferSize
 	}
 
-	// If there is no network_config, assume it is an old version of the config, and enable
-	// the network check. This has the downside that if the network_config is deleted, network
-	// check will be enabled.  However, this seems like the best option for backwards compatibility.
-	//
-	// The network check won't be run if sysprobe isn't enabled so there's no need to check
-	// EnableSystemProbe here.
-	networkEnabled := true
-	// network_config is a top-level config, not a subconfig of system_probe_config
-	if config.Datadog.IsSet("network_config") {
-		// System probe can be run without the network module as determined on the network_config.enabled value
-		networkEnabled = config.Datadog.GetBool("network_config.enabled")
-		log.Info(fmt.Sprintf("network_config found, enabled = %v", networkEnabled))
-	} else {
-		log.Info("network_config not found, enabling network check by default")
-	}
-	if networkEnabled {
-		a.EnabledChecks = append(a.EnabledChecks, "Network")
+	// Enable network and connections check
+	if config.Datadog.GetBool("network_config.enabled") {
+		log.Info(fmt.Sprintf("network_config found.enabled found, enabling system-probe with network module enabled"))
```

Comment:
Grammar seems a little odd here, maybe:

> network_config found, enabled parameter set to true: enabling system-probe with network module running.

Reply:
I changed it to `network_config.enabled was true: enabling system-probe with network module running.`